### ISSUE TITLE
Data: aa sequences are missing for some CDSs

### ISSUE THREAD

**Issue by mshukla1** [PATRIC3/patric3_website#1289]
AA sequences are missing for ~130,000 CDSs annotated by PATRIC. 
They belong to ~11,000 genomes in to total. 

Actions: 
- Add missing aa sequences
- Update corresponding faa files

**Comment by mshukla1** [comment]
After further analysis, it seems most of the CDSs missing AA sequences fall in one of the following categories: 
- partial CDSs on the contig boundaries
- contain ambiguous bases
- correspond to pseudogenes in refseq annotations  

All of the above situations result in erroneous translation resulting in uncommon start and/or premature stop codons. Hence, I decided not to populate those aa sequences in the database. 

I have updated script that generate download files to exclude such features, to avoid fasta entries without sequences. 

-Maulik
